### PR TITLE
new-feature:add k3http

### PR DESCRIPTION
**Pull request by WRY-learning**
The k3http modified according to the specification

### CODE REVIEW

File: test/test_k3http.py [hunk 1]
```diff
@@ -2,14 +2,14 @@
 import unittest
```
Comment (drmingdrmer) [inline test/test_k3http.py:2]: Why keep this file? it should be a template. 🤔

File: test/test_httpclient.py [hunk 2]
```diff
@@ -0,0 +1,623 @@
+#!/usr/bin/env python2
```
Comment (drmingdrmer) [inline test/test_httpclient.py:1]: Remove this, or update it to python3.

File: stopwatch.py [hunk 3]
```diff
@@ -0,0 +1,384 @@
+"""StopWatch - library for adding timers and tags in your code for performance monitoring
```
Comment (drmingdrmer) [inline stopwatch.py:1]: stopwatch should be standalone script that could be used by all pykit package.

File: client.py [hunk 4]
```diff
@@ -0,0 +1,573 @@
+#!/usr/bin/env python2
```
Comment (drmingdrmer) [inline client.py:1]: python3 please...

**Review by IloveKanade — COMMENTED** [review]
There's no need to keep this file, just delete it

File: test/test_k3http.py [hunk 5]
```diff
@@ -2,14 +2,14 @@
...
 
 import k3ut
-import {{ name }}
+import k3http
 
 dd = k3ut.dd
 
 this_base = os.path.dirname(__file__)
```
Comment (IloveKanade) [inline test/test_k3http.py:9]: There's no need to keep this file, just delete it

**Review by drmingdrmer — APPROVED** [review]
LGTM~

**Pull request merged**